### PR TITLE
Add .net 6.0 p1 test suite

### PR DESCRIPTION
**Pull request by vtikoo**
Signed-off-by: Vikas Tikoo <vitikoo@microsoft.com>

### CODE REVIEW

File: tests/coreclr/p1/Makefile [hunk 1]
```diff
@@ -0,0 +1,121 @@
...
+	# # This test returns error code for non-windows systems, by design.
+	$(RUN_SINGLE_TEST) $(MYST_EXEC) config_256m.json 300 ext2 \
+	Interop/ExecInDefAppDom/ExecInDefAppDom/ExecInDefAppDom.dll 111
+
+	# # Change CWD using mystikos config for the following tests to run successfully
+	sed -i '19 i \ \ \ \ "CurrentWorkingDirectory":"/coreclr-tests-all/Interop/COM/Reflection/Reflection",' config_1g.json
+	$(RUN_SINGLE_TEST) $(MYST_EXEC) config_1g.json 300 ext2 \
+	Interop/COM/Reflection/Reflection/Reflection.dll
+	sed -i '19d' config_1g.json
+	sed -i '19 i \ \ \ \ "CurrentWorkingDirectory":"/coreclr-tests-all/JIT/Regression/JitBlue/DevDiv_461649/DevDiv_461649",' config_1g.json
```
Comment (jxyang) [inline tests/coreclr/p1/Makefile:49]: What are the "sed" commands for? Why can't we put `CurrentWorkingDirectory` in the config file?

File: tests/coreclr/p1/Makefile [hunk 2]
```diff
@@ -0,0 +1,121 @@
...
+	@ $(MAKE) -s run
+
+##################################
+#         dev targets            #
+##################################
+
+one:
+	$(MYST_EXEC) ext2fs $(OPTS) \
+	/coreclr-tests-all/Tests/Core_Root/corerun \
+	/coreclr-tests-all/$(TEST_CASE)
```
Comment (jxyang) [inline tests/coreclr/p1/Makefile:113]: Maybe change to `TEST` to match ltp and libcxx test suites?

File: tests/coreclr/p1/Makefile (hunk 1 above)
Comment (vtikoo) [reply tests/coreclr/p1/Makefile:49]: This was added while enabling the p1 suite. cc: @RRathna @bodzhang

**Comment by jxyang** [comment]
Please adopt changes in https://github.com/deislabs/mystikos/pull/1165 to add tests that require extra command line parameters, either in this PR or a follow up PR.

**Review by jxyang — APPROVED** [review]

File: tests/coreclr/p1/Makefile (hunk 2 above)
Comment (vtikoo) [reply tests/coreclr/p1/Makefile]: Done

**Comment by vtikoo** [comment]
Superseded by #1170.

**Pull request closed without merging**